usrsctplib/user_recv_thread.c

```diff
@@ -545,7 +548,7 @@ recv_function_raw6(void *arg)
 		msg.msg_controllen = (socklen_t)CMSG_SPACE(sizeof (struct in6_pktinfo));
 		msg.msg_flags = 0;
 
-		ncounter = n = recvmsg(SCTP_BASE_VAR(userspace_rawsctp6), &msg, 0);
+		ncounter = n = (int)recvmsg(SCTP_BASE_VAR(userspace_rawsctp6), &msg, 0);
```

Comment:
Fixed in https://github.com/sctplab/usrsctp/commit/a8adc4ae7cdfd014637ae3c0caf55d2bfda5ffc5